WordPress/src/main/java/org/wordpress/android/util/AnalyticsUtils.java

```diff
@@ -218,15 +220,25 @@ public static void trackWithReaderPostDetails(AnalyticsTracker.Stat stat, long b
         trackWithReaderPostDetails(stat, ReaderPostTable.getBlogPost(blogId, postId, true));
     }
 
-    public static void trackWithBlogPostDetails(AnalyticsTracker.Stat stat, long blogId, long postId) {
+    public static void trackWithBlogPostDetails(AnalyticsTracker.Stat stat, String blogId, String postId) {
```

Comment:
These were originally coded as `long` values that were the actual blogId and postId, but now they're strings which can be slugs instead of IDs.  Is there any chance this will confuse analytics reporting, because IDs rather than slugs were expected?